app/src/main/java/org/android/go/sopt/ui/gallery/GalleryFragment.kt

```diff
@@ -22,4 +30,21 @@ class GalleryFragment : Fragment() {
             DataBindingUtil.inflate(inflater, R.layout.fragment_gallery, container, false)
         return binding.root
     }
-}
\ No newline at end of file
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.btnGalleryPickImage.setOnClickListener {
+            launcher.launch(PickVisualMediaRequest(ActivityResultContracts.PickVisualMedia.ImageAndVideo))
+            viewModel.image.observe(viewLifecycleOwner) {
+
+                viewModel.uploadImage(it.toFormData())
+            }
```

Comment:
```suggestion
            viewModel.image.observe(viewLifecycleOwner) { image ->
                viewModel.uploadImage(image.toFormData())
            }
```
스코프 함수에서 `it` 대신 명시적인 네이밍을 써주면 가독성이 좋아질 것 같습니다!